basestar-graphql/src/test/resources/io/basestar/graphql/schema.yml

```diff
@@ -123,3 +123,20 @@ Function:
   returns: boolean
   definition: ""
 
+NearPoints:
+  type: query
+  sql: SELECT X, Y FROM ${point}
```

Comment:
The query does not seem to be making use of the arguments

Reply:
at this level of test we're only testing the GQL schema adaptor, the whole storage layer is mocked out, so doesn't really matter what the SQL is